CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 2.36.0-rc0 / 2022-05-25
+
+This Prometheus release uses go 1.18 and needs at least go1.17 going forward.
+
+* [FEATURE] Add lowercase and uppercase relabel action. #10641
+* [FEATURE] SD: Add IONOS Cloud integration. #10514
+* [FEATURE] SD: Add Vultr integration. #10714
+* [FEATURE] SD: Add Linode SD failure count metric. #10673
+* [FEATURE] Add prometheus_ready metric. #10682
+* [ENHANCEMENT] Add stripDomain to template function. #10475
+* [ENHANCEMENT] UI: Enable active search through dropped targets. #10668
+* [ENHANCEMENT] Stop rule manager before TSDB is stopped. #10680
```

Comment:
IMHO this is more of a `BUGFIX` for me since previous releases would trigger false positive alerts

Reply:
Updated. :+1: